plugin/pkg/scheduler/util/utils.go

```diff
@@ -28,33 +27,115 @@ import (
 
 const DefaultBindAllHostIP = "0.0.0.0"
 
-// GetUsedPorts returns the used host ports of Pods: if 'port' was used, a 'port:true' pair
+type ProtocolPort struct {
```

Comment:
Can you add comments here for both functions and types. The changes look good. Wondering why golint isn't complaining.

Reply:
Is it true that golint is enforced in CI?

Reply:
right, I don't think CI checks it.. will add them.